forastero/bench.py

```diff
@@ -51,11 +51,17 @@ class BaseBench:
 
     TEST_REQ_PARAMS: ClassVar[dict[str, set[str]]] = defaultdict(set)
     PARAM_FILE_PATH: ClassVar[str] = os.environ.get("TEST_PARAMS", None)
-    PARSED_PARAMS: ClassVar[dict[str, Any]] = (
-        json.loads(Path(PARAM_FILE_PATH).read_text(encoding="utf-8"))
-        if PARAM_FILE_PATH
-        else {}
-    )
+    PARSED_PARAMS: ClassVar[dict[str, Any]] = {
+        k.lower().strip(): v for k, v in
+        (json.loads(Path(PARAM_FILE_PATH).read_text(encoding="utf-8"))
+         if PARAM_FILE_PATH
+         else {
+             # Control logging verbosity
+             "verbosity": "info",
+             # Enable profiling by providing a path
+             "profiling": None,
+         }).items()
+    }
```

Comment:
This ternary is pretty complicated... any reason this can't be a classmethod with memo ?

Reply:
Changed to `get_parameters()`